editor/components/block-edit/context.js

```diff
@@ -10,6 +10,8 @@ import { createContext, createHigherOrderComponent } from '@wordpress/element';
 
 const { Consumer, Provider } = createContext( {
 	name: '',
+	attributes: {},
+	setAttribute: noop,
```

Comment:
We always used the setAttributes "concept", maybe here we can expose setAttributes, so we are not exposing a new concept (setAttribute).

Reply:
I don't have any preference here, I can update to `setAttributes` if it makes things easier.